tests/scripts/dind-cluster.sh

```diff
@@ -180,18 +183,20 @@ function dind::check-image {
 
 function dind::filter-make-output {
   # these messages make output too long and make Travis CI choke
-  egrep -v --line-buffered 'I[0-9][0-9][0-9][0-9] .*(parse|conversion|defaulter|deepcopy)\.go:[0-9]+\]'
+   grep -E -v --line-buffered 'I[0-9][0-9][0-9][0-9] .*(parse|conversion|defaulter|deepcopy)\.go:[0-9]+\]'
 }
 
 function dind::run-build-command {
     # this is like build/run.sh, but it doesn't rsync back the binaries,
     # only the generated files.
+    # shellcheck disable=SC2034 # Var is exported
     local cmd=("$@")
```

Comment:
If it's unused, then we don't need it. OR, on the line `kube::build::run_build_command "$@"` use the command array instead. I would just remove the `local cmd=("$@")` line here.